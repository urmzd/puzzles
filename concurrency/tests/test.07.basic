Adding transaction: 11235 A012345678901234567890123456 B012345678901234567890123456 150000 20
Adding transaction: 12358 C012345678901234567890123456 D012345678901234567890123456 150000 70
Adding transaction: 35813 E012345678901234567890123456 F012345678901234567890123456 150000 140
Adding transaction: 58132 G012345678901234567890123456 H012345678901234567890123456 150000 210
Adding transaction: 81321 I012345678901234567890123456 J012345678901234567890123456 150000 280
Adding transaction: 13213 K012345678901234567890123456 L012345678901234567890123456 150000 350
Adding transaction: 32134 M N 150000 10
Adding transaction: 21345 O P 150000 15
Block mined: 1 0 0x00000000 4
13213 K012345678901234567890123456 L012345678901234567890123456 150000 350
81321 I012345678901234567890123456 J012345678901234567890123456 150000 280
58132 G012345678901234567890123456 H012345678901234567890123456 150000 210
32134 M N 150000 10
0x0000015c 0x00d8c42c
Block mined: 2 1 0x00d8c42c 4
35813 E012345678901234567890123456 F012345678901234567890123456 150000 140
12358 C012345678901234567890123456 D012345678901234567890123456 150000 70
11235 A012345678901234567890123456 B012345678901234567890123456 150000 20
21345 O P 150000 15
0x00000467 0x008a857b
