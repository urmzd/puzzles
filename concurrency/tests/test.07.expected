Received event TRX with ID 11235
Received event TRX with ID 11236
Received event TRX with ID 11237
Received event TRX with ID 11238
Received event TRX with ID 11239
Received event TRX with ID 11240
Received event TRX with ID 11241
Received event TRX with ID 11242
Received event TRX with ID 11243
Adding transaction: 11235 Al_Alpha Beth_Beta 150000 40
Adding transaction: 11236 Cat_Charlie Dov_Delta 150000 20
Adding transaction: 11237 Eve_Echo Fin_Foxtrot 150000 25
Adding transaction: 11238 Guy_Golf Hana_Hotel 150000 40
Adding transaction: 11239 Iona_Indigo Jeff_Juliet 150000 20
Adding transaction: 11240 Kit_Kilo Liam_Lima 150000 25
Adding transaction: 11241 Mona_Mike Nick_November 150000 40
Adding transaction: 11242 Otto_Oscar Peta_Papa 150000 20
Adding transaction: 11243 Qu_Quebec Reya_Romeo 150000 25
Received event MINE
Received event END
Block mined: 1 0 0x00000000 6
11235 Al_Alpha Beth_Beta 150000 40
11238 Guy_Golf Hana_Hotel 150000 40
11241 Mona_Mike Nick_November 150000 40
11236 Cat_Charlie Dov_Delta 150000 20
11237 Eve_Echo Fin_Foxtrot 150000 25
11239 Iona_Indigo Jeff_Juliet 150000 20
Thread 0 checking nonce 0x00000000
Thread 0 checking nonce 0x00000008
Thread 0 checking nonce 0x00000010
Thread 0 checking nonce 0x00000018
Thread 0 checking nonce 0x00000020
Thread 0 checking nonce 0x00000028
Thread 0 checking nonce 0x00000030
Thread 0 checking nonce 0x00000038
Thread 0 checking nonce 0x00000040
Thread 0 checking nonce 0x00000048
Thread 0 checking nonce 0x00000050
Thread 1 checking nonce 0x00000001
Thread 1 checking nonce 0x00000009
Thread 1 checking nonce 0x00000011
Thread 1 checking nonce 0x00000019
Thread 1 checking nonce 0x00000021
Thread 1 checking nonce 0x00000029
Thread 1 checking nonce 0x00000031
Thread 1 checking nonce 0x00000039
Thread 1 checking nonce 0x00000041
Thread 1 checking nonce 0x00000049
Thread 1 checking nonce 0x00000051
Thread 2 checking nonce 0x00000002
Thread 2 checking nonce 0x0000000a
Thread 2 checking nonce 0x00000012
Thread 2 checking nonce 0x0000001a
Thread 2 checking nonce 0x00000022
Thread 2 checking nonce 0x0000002a
Thread 2 checking nonce 0x00000032
Thread 2 checking nonce 0x0000003a
Thread 2 checking nonce 0x00000042
Thread 2 checking nonce 0x0000004a
Thread 2 checking nonce 0x00000052
Thread 4 checking nonce 0x00000004
Thread 4 checking nonce 0x0000000c
Thread 4 checking nonce 0x00000014
Thread 4 checking nonce 0x0000001c
Thread 4 checking nonce 0x00000024
Thread 4 checking nonce 0x0000002c
Thread 4 checking nonce 0x00000034
Thread 4 checking nonce 0x0000003c
Thread 4 checking nonce 0x00000044
Thread 3 checking nonce 0x00000003
Thread 3 checking nonce 0x0000000b
Thread 3 checking nonce 0x00000013
Thread 3 checking nonce 0x0000001b
Thread 3 checking nonce 0x00000023
Thread 3 checking nonce 0x0000002b
Thread 3 checking nonce 0x00000033
Thread 3 checking nonce 0x0000003b
Thread 3 checking nonce 0x00000043
Thread 3 checking nonce 0x0000004b
Thread 5 checking nonce 0x00000005
Thread 5 checking nonce 0x0000000d
Thread 5 checking nonce 0x00000015
Thread 5 checking nonce 0x0000001d
Thread 5 checking nonce 0x00000025
Thread 5 checking nonce 0x0000002d
Thread 5 checking nonce 0x00000035
Thread 5 checking nonce 0x0000003d
Thread 5 checking nonce 0x00000045
Thread 6 checking nonce 0x00000006
Thread 6 checking nonce 0x0000000e
Thread 6 checking nonce 0x00000016
Thread 6 checking nonce 0x0000001e
Thread 6 checking nonce 0x00000026
Thread 6 checking nonce 0x0000002e
Thread 6 checking nonce 0x00000036
Thread 6 checking nonce 0x0000003e
Thread 6 checking nonce 0x00000046
Thread 7 checking nonce 0x00000007
Thread 7 checking nonce 0x0000000f
Thread 7 checking nonce 0x00000017
Thread 7 checking nonce 0x0000001f
Thread 7 checking nonce 0x00000027
Thread 7 checking nonce 0x0000002f
Thread 7 checking nonce 0x00000037
Thread 7 checking nonce 0x0000003f
Thread 7 checking nonce 0x00000047
0x00000044 0x0031353d
